MSAL/test/integration/native_auth/end_to_end/sign_in/MSALNativeAuthSignInUserNameAndPasswordEndToEndTests.swift

```diff
@@ -115,4 +115,82 @@ final class MSALNativeAuthSignInUsernameAndPasswordEndToEndTests: MSALNativeAuth
         XCTAssertTrue(signInPasswordRequiredDelegateSpy.onSignInPasswordRequiredErrorCalled)
         XCTAssertEqual(signInPasswordRequiredDelegateSpy.error?.isInvalidPassword, true)
     }
+    
+    // Sign In - Verify Custom URL Domain - "https://<tenantName>.ciamlogin.com/<tenantName>.onmicrosoft.com"
+    func test_signInCustomDomain1InSuccess() async throws {
+        guard let sut = initialisePublicClientApplication(customSubdomainFormat: 0) else {
+            XCTFail("Failed to initialise auth client")
+            return
+        }
+
+        guard let username = retrieveUsernameForSignInUsernameAndPassword(), let password = await retrievePasswordForSignInUsername() else {
+            XCTFail("Missing information")
+            return
+        }
+
+        let signInExpectation = expectation(description: "Signing in")
+        let signInDelegateSpy = SignInPasswordStartDelegateSpy(expectation: signInExpectation)
+
+        // Perform the sign-in asynchronously
+        Task {
```

Comment:
Hi Yuki, why did you call the signIn function in a Task block and using "await"?

Reply:
Remove the await and use the same style as test_signInUsingPasswordWithKnownUsernameResultsInSuccess. Done.